homeassistant/components/qnap_qsw/coordinator.py

```diff
@@ -30,13 +31,14 @@ def __init__(self, hass: HomeAssistant, qsw: QnapQswApi) -> None:
             _LOGGER,
             name=DOMAIN,
             update_interval=SCAN_INTERVAL,
+            update_method=self._async_update,
         )
 
-    async def _async_update_data(self):
+    async def _async_update(self) -> dict[str, Any]:
```

Comment:
Why did we change from `_async_update_data`?

Reply:
Hi @MartinHjelmare, this is the same as https://github.com/home-assistant/core/pull/71604#discussion_r868451875.
I will open a new PR and fix this.

Reply:
New PR: https://github.com/home-assistant/core/pull/71628